identity-apps-core/components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/internal/AppsCommonDataHolder.java

```diff
@@ -125,4 +131,40 @@ public void setOrganizationManagementEnabled(
             isOrganizationManagementEnabled = organizationManagementInitializeService.isOrganizationManagementEnabled();
         }
     }
+
+    /**
+     * Get organization manager.
+     * @return OrganizationManager.
+     */
+    public OrganizationManager getOrganizationManager() {
+
+        return organizationManager;
+    }
+
+    /**
+     * Set organization manager.
+     * @param organizationManager OrganizationManager.
+     */
+    public void setOrganizationManager(OrganizationManager organizationManager) {
+
+        this.organizationManager = organizationManager;
+    }
+
+    /**
+     * Get realm service.
+     * @return RealmService.
+     */
+    public RealmService getRealmService() {
+
+        return realmService;
+    }
+
+    /**
+     * Set realm service.
+     * @param realmService RealmService.
+     */
```

Comment:
comments' has the formatting issue as mentioned in https://github.com/wso2/identity-apps/pull/4216/files#r1359373388